config/config.go

```diff
@@ -14,21 +14,22 @@ import (
 )
 
 type ICEConfig struct {
-	Username     string           `yaml:"username,omitempty"`
-	Password     string           `yaml:"password,omitempty"`
-	ApiKey       string           `json:"apiKey,omitempty" yaml:"api_key,omitempty"`
-	AccountSid   string           `yaml:"account_sid,omitempty"`
-	RequestUrl   string           `json:"requestUrl,omitempty" yaml:"request_url,omitempty"`
-	HttpUsername string           `yaml:"http_username"`
-	HttpPassword string           `yaml:"http_password"`
-	Enabled      bool             `yaml:"enabled"`
-	StunHost     string           `yaml:"stun_host,omitempty"`
-	TurnHost     string           `yaml:"turn_host,omitempty"`
-	TurnPorts    map[string][]int `yaml:"turn_ports,omitempty"`
-	StunPorts    map[string][]int `yaml:"stun_ports,omitempty"`
-	StunEnabled  bool             `yaml:"stun_enabled"`
-	TurnEnabled  bool             `yaml:"turn_enabled"`
-	DoThroughput bool             `yaml:"do_throughput"`
+	Username      string           `yaml:"username,omitempty"`
+	Password      string           `yaml:"password,omitempty"`
+	ApiKey        string           `json:"apiKey,omitempty" yaml:"api_key,omitempty"`
+	AccountSid    string           `yaml:"account_sid,omitempty"`
+	RequestUrl    string           `json:"requestUrl,omitempty" yaml:"request_url,omitempty"`
+	HttpUsername  string           `yaml:"http_username"`
+	HttpPassword  string           `yaml:"http_password"`
+	Enabled       bool             `yaml:"enabled"`
+	UseStunRfcUri bool             `yaml:"use_stun_rfc_uri"`
```

Comment:
I think this should be rfc specific?

Reply:
makes sense. perhaps something like this?
```suggestion
	UseStunRfc7094Uri bool             `yaml:"use_stun_rfc7094_uri"`
```

Reply:
I'm good with that..... Elsewhere in the config we use URI all uppercase.... I've no idea what is _right_ Should RFC all be upper case too? Should URI be upper case to follow convention elsewhere? Sorry!

Reply:
New version in cd42f7c: 
```go
StunUseRFC7094URI bool             `yaml:"stun_use_rfc7094_uri"`
```

wdyt?

Reply:
I think thats consistent :)